packages/core/src/bus/Decorators/Bind.ts

```diff
@@ -0,0 +1,8 @@
+import 'reflect-metadata';
+import { Event, EventType } from '../Event';
+
+export function bind<T>(event: EventType<T>): ClassDecorator {
+  return target => {
+    Reflect.defineMetadata(Event, event, target);
+  };
+}
```

Comment:
The preferred way is to use the arrow function. Moreover, according to LLD, you should be able to bind multiple events to the same handler.

```suggestion
export const bind =
  (...events: EventType[]): ClassDecorator =>
  (target) => {
    Reflect.defineMetadata(Event, events, target);
  };

```